test/service/test_response.py

```diff
@@ -2035,24 +2042,50 @@ def test_summary_filter_none(self):
 
 class TestUnpopulatedIndexResponse(WebServiceTestCase):
 
+    @classmethod
+    def bundles(cls) -> List[BundleFQID]:
+        return []
+
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.index_service.create_indices(cls.catalog)
+        cls._setup_indices()
+        cls.maxDiff = None
 
     @classmethod
     def tearDownClass(cls):
-        cls.index_service.delete_indices(cls.catalog)
+        cls._teardown_indices()
         super().tearDownClass()
 
+    def facets(self) -> Sequence[str]:
+        return self.app_module.app.facets
+
+    entity_types = ['files', 'projects', 'samples', 'bundles']
+
     def test_empty_response(self):
-        url = self.base_url + "/index/projects"
-        response = requests.get(url)
-        response.raise_for_status()
-        response = response.json()
-        self.assertEqual([], response['hits'])
-        self.assertEqual({None}, set(response['pagination'].values()))
-        self.assertEqual({}, response['termFacets'])
+        empty_term_facets = {
+            facet: {'terms': [], 'total': 0, 'type': 'terms'} for facet in self.facets()
```

Comment:
line too long

Reply:
I can change it, but it should meet the trimmed line length requirements as it's 76 char.